cnquery panics on `asset{*}` query

**Describe the bug**
When I run cnquery with `asset{*}` as query it panics.

**To Reproduce**
Steps to reproduce the behavior:

1. Run `cnquery run k8s -c 'asset{ * }'`
2. Note the error

**Expected behavior**
I would expect valid output and no panic.

**Screenshots or CLI Output**
```
cnquery run k8s -c 'asset{ * }' 
→ loaded configuration from /etc/opt/mondoo/mondoo.yml using source default
→ discover related assets for 1 asset(s)
→ discovery option auto is used. This will detect the assets: cluster, cronjobs, daemonsets, deployments, ingresses, jobs, pods, replicasets, statefulsets
→ use cluster name from kube config cluster-name=minikube
→ resolved assets resolved-assets=11
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x28 pc=0x7645f4]

goroutine 216 [running]:
net/http.(*Client).deadline(0x203000?)
	/home/runner/_work/_tool/go/1.19.7/x64/src/net/http/client.go:188 +0x14
net/http.(*Client).do(0x0, 0xc0006ed600)
	/home/runner/_work/_tool/go/1.19.7/x64/src/net/http/client.go:599 +0x225
net/http.(*Client).Do(0xee7fcc0?, 0xc0000be0ff?)
	/home/runner/_work/_tool/go/1.19.7/x64/src/net/http/client.go:581 +0x19
go.mondoo.com/ranger-rpc.(*Client).DoClientRequest(0xc001438440, {0xa800ba8?, 0xc0000be0b0}, {0xa758ec0, 0x0}, {0xc00095e700, 0x36}, {0xa754420, 0xc0005dbc80}, {0xa7545a0, ...})
	/home/runner/go/pkg/mod/go.mondoo.com/ranger-rpc@v0.0.0-20230328135530-12135c17095f/client.go:86 +0x5b6
go.mondoo.com/cnquery/upstream/mvd.(*AdvisoryScannerClient).AnalyseAsset(0xc001438440, {0xa800ba8, 0xc0000be0b0}, 0xc0005dbc80?)
	/home/runner/_work/cnquery/cnquery/upstream/mvd/mvd.ranger.go:54 +0xf7
go.mondoo.com/cnquery/resources/packs/core.(*mqlAsset).GetVulnerabilityReport(0x0?)
	/home/runner/_work/cnquery/cnquery/resources/packs/core/asset_advisories.go:107 +0x50f
go.mondoo.com/cnquery/resources/packs/core.(*mqlAsset).ComputeVulnerabilityReport(0xc000948d80)
	/home/runner/_work/cnquery/cnquery/resources/packs/core/core.lr.go:1317 +0x55
go.mondoo.com/cnquery/resources/packs/core.(*mqlAsset).MqlCompute(0xc001438200?, {0xc000906870, 0x13})
	/home/runner/_work/cnquery/cnquery/resources/packs/core/core.lr.go:1291 +0x326
go.mondoo.com/cnquery/resources.(*Runtime).WatchAndUpdate(0xc001438240, {0xa806a60, 0xc000948d80}, {0xc000906870, 0x13}, {0xc00095e5c0, 0x31}, 0xc000ed2120)
	/home/runner/_work/cnquery/cnquery/resources/runtime.go:350 +0x39b
go.mondoo.com/cnquery/llx.runResourceFunction(0xc000353030, 0xc00162bc20, 0xc0019fe640, 0x20000000e)
	/home/runner/_work/cnquery/cnquery/llx/builtin.go:712 +0x23e
go.mondoo.com/cnquery/llx.(*blockExecutor).runBoundFunction(0xc000353030, 0xc00162bc20, 0xc0019fe640, 0x4?)
	/home/runner/_work/cnquery/cnquery/llx/builtin.go:795 +0xf2
go.mondoo.com/cnquery/llx.(*blockExecutor).runFunction(0xc000353030, 0xc0019fe640, 0x0?)
	/home/runner/_work/cnquery/cnquery/llx/llx.go:764 +0xde
go.mondoo.com/cnquery/llx.(*blockExecutor).runChunk(0xc000353030, 0xc00162bfb0?, 0x20000000e?)
	/home/runner/_work/cnquery/cnquery/llx/llx.go:781 +0x25e
go.mondoo.com/cnquery/llx.(*blockExecutor).runRef(0xc0008b3560?, 0x6?)
	/home/runner/_work/cnquery/cnquery/llx/llx.go:806 +0xef
go.mondoo.com/cnquery/llx.(*blockExecutor).runChain(0xc000353030, 0x958c0b3?)
	/home/runner/_work/cnquery/cnquery/llx/llx.go:838 +0xa6
go.mondoo.com/cnquery/llx.(*blockExecutor).run(0xc000353030)
	/home/runner/_work/cnquery/cnquery/llx/llx.go:351 +0x2d7
go.mondoo.com/cnquery/llx.(*blockExecutor).runFunctionBlock(0xc000352f50, {0xc000948f00, 0x1, 0xc00162bd40?}, 0x200000000, 0x0?)
	/home/runner/_work/cnquery/cnquery/llx/llx.go:582 +0x12f
go.mondoo.com/cnquery/llx.(*blockExecutor).runFunctionBlocks(0xc000352f50, {0xc0016a5880, 0x1, 0xc000a57800?}, 0x9580610?, 0x2?)
	/home/runner/_work/cnquery/cnquery/llx/llx.go:553 +0x11b
go.mondoo.com/cnquery/llx.(*blockExecutor).runBlock(0xc000352f50, 0xc00162bc20, 0x9a31d78?, {0xc000948cb8, 0x0, 0x0?}, 0x100000002)
	/home/runner/_work/cnquery/cnquery/llx/llx.go:617 +0x40d
go.mondoo.com/cnquery/llx.init.0.func1(0xc00070a850?, 0x6?, 0x9580610?, 0x2?)
	/home/runner/_work/cnquery/cnquery/llx/builtin.go:646 +0x52
go.mondoo.com/cnquery/llx.(*blockExecutor).runBoundFunction(0xc000352f50, 0xc00162bc20, 0xc0019fe690, 0xa?)
	/home/runner/_work/cnquery/cnquery/llx/builtin.go:782 +0x119
go.mondoo.com/cnquery/llx.(*blockExecutor).runFunction(0xc000352f50, 0xc0019fe690, 0x203000?)
	/home/runner/_work/cnquery/cnquery/llx/llx.go:764 +0xde
go.mondoo.com/cnquery/llx.(*blockExecutor).runChunk(0xc000352f50, 0xc00162b530?, 0x100000002?)
	/home/runner/_work/cnquery/cnquery/llx/llx.go:781 +0x25e
go.mondoo.com/cnquery/llx.(*blockExecutor).runRef(0xc0008b31a0?, 0xc00162b500?)
	/home/runner/_work/cnquery/cnquery/llx/llx.go:806 +0xef
go.mondoo.com/cnquery/llx.(*blockExecutor).runChain(0xc000352f50, 0x958c0b3?)
	/home/runner/_work/cnquery/cnquery/llx/llx.go:838 +0xa6
go.mondoo.com/cnquery/llx.(*blockExecutor).run(0xc000352f50)
	/home/runner/_work/cnquery/cnquery/llx/llx.go:351 +0x2d7
go.mondoo.com/cnquery/llx.(*MQLExecutorV2).Run(0xc0005db5c0?)
	/home/runner/_work/cnquery/cnquery/llx/llx.go:283 +0x59
go.mondoo.com/cnquery/mql/internal.(*executionManager).executeCodeBundle(0xc0019fe6e0, 0xc00039bea0, 0x0?, {0x0, 0x0})
	/home/runner/_work/cnquery/cnquery/mql/internal/execution_manager.go:167 +0x35d
go.mondoo.com/cnquery/mql/internal.(*executionManager).Start.func1()
	/home/runner/_work/cnquery/cnquery/mql/internal/execution_manager.go:85 +0x1cb
created by go.mondoo.com/cnquery/mql/internal.(*executionManager).Start
	/home/runner/_work/cnquery/cnquery/mql/internal/execution_manager.go:54 +0x6a
```

**Desktop (please complete the following information):**

- cnquery v8.9.0

**Additional context**
This also happens for `local`: `cnquery run local -c 'asset{ * }'`

This also happens for this query: `cnquery run local -c 'asset.vulnerabilityReport'`

But running the query inside a shell works:
```
cnquery shell local 
→ loaded configuration from /etc/opt/mondoo/mondoo.yml using source default
→ discover related assets for 1 asset(s)
→ resolved assets resolved-assets=1
  ___ _ __   __ _ _   _  ___ _ __ _   _ 
 / __| '_ \ / _` | | | |/ _ \ '__| | | |
| (__| | | | (_| | |_| |  __/ |  | |_| |
 \___|_| |_|\__, |\__,_|\___|_|   \__, |
  mondoo™      |_|                |___/  interactive shell

cnquery> asset.vulnerabilityReport
asset.vulnerabilityReport: {
  advisories: [
    0: {
      ID: "USN-5958-1"
      Mrn: "//vadvisor.api.mondoo.app/advisories/USN-5958-1"
...
```

--- a/apps/cnquery/cmd/plugin.go
+++ b/apps/cnquery/cmd/plugin.go
@@ -137,6 +137,7 @@ func (c *cnqueryPlugin) RunQuery(conf *proto.RunQueryConfig, out shared.OutputHe
 			SpaceMrn:    opts.GetParentMrn(),
 			ApiEndpoint: opts.UpstreamApiEndpoint(),
 			Plugins:     []ranger.ClientPlugin{certAuth},
+			HttpClient:  ranger.DefaultHttpClient(),
 		}
 	}
 

